WordPress/gutenberg
# Comments Pagination Numbers : Add border and spacing support

## What?
Add border block support to the comments pagination numbers block.
Part of https://github.com/WordPress/gutenberg/issues/43247
Part of https://github.com/WordPress/gutenberg/issues/43243

## Why?
The comments pagination numbers block is missing border support.

## How?
Adds the border block support in block.json

## Testing Instructions

1. Go to the Global Styles setting ( under appearance > editor > styles > edit styles > blocks )
2. Make sure that the comments pagination numbers block's border and spacing are configurable via Global Styles
3. Verify that Global Styles are applied correctly in the editor and frontend
4. Edit the template/page, Add a comments pagination numbers block and Apply the border and spacing styles
5. Verify that block styles take precedence over global styles
6. Verify that block borders and spacing display correctly in both the editor and frontend


## Screenshots or screencast 
<img width="1433" alt="comments-numbers" src="https://github.com/user-attachments/assets/4ecdb7f3-d3f5-4f08-94d2-a0cfc79ce41d">
